index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Harmattan&family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/9947fbd823.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header class="template5-main-mobile">
+      <div class="app-bar">
+        <div class="time"><img src="Time.png" alt="time" /></div>
+        <a href="#" class="name">Gidiz Logo</a>
+      </div>
+      <i class="fa-solid fa-bars"></i>
+      <div class="ellipse1"></div>
+      <div class="ellipse2"></div>
+
+      <section class="headline">
+        <h1 class="primary-text">Gideon Dadi</h1>
+        <h3 class="supporting-text">Software Engineer</h3>
+
+        <div class="advert-text">
+          <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <hr />
+      </section>
+      <section class="section-2">
+        <div class="language">
+          <h2>Language</h2>
+          <p>Javascript</p>
+          <p>Ruby</p>
+          <p>Node</p>
+          <p>CSS</p>
+          <p>Html</p>
+        </div>
+
+        <div class="framewoks">
+          <h2>Frameworks</h2>
+          <p>Bootstrap</p>
+          <p>Ruby ob Rails</p>
+          <p>RSPec</p>
+          <p>CapyBara</p>
+          <p>Selenium</p>
+        </div>
+
+        <div class="skills">
+          <h2>Skills</h2>
+          <p>Codekit</p>
+          <p>GitHub</p>
+          <p>Codepen</p>
+          <p>Gitlab</p>
+          <p>Terminal</p>
+        </div>
+        <div class="vector5"></div>
+      </section>
+
+      <div class="icons">
+        <i class="fa-brands fa-github"></i>
+        <i class="fab fa-linkedin-in"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fab fa-medium"></i>
+      </div>
+    </header>
+  </body>
```

Comment:
- [ ] Please, consider implementing the icon for scrolling as in the template:
<img width="292" alt="Screen Shot 2022-02-15 at 18 12 31" src="https://user-images.githubusercontent.com/49064106/154079112-91d7d8fb-817b-4e48-a037-16f7ea40d481.png">

Reply:
Noted -Please the link is in the work session which is not to in this project.